src/inference/include/openvino/runtime/properties.hpp

```diff
@@ -784,6 +784,21 @@ inline std::istream& operator>>(std::istream& is, CacheMode& mode) {
  */
 static constexpr Property<CacheMode, PropertyMutability::RW> cache_mode{"CACHE_MODE"};
 
+struct EncryptionCallbacks {
+    std::function<std::string(const std::string&)> encrypt;
+    std::function<std::string(const std::string&)> decrypt;
+};
+
+/**
+ * @brief Write-only property to set encryption/decryption function for saving/loading model cache.
+ * If cache_crypto_callback is set, the model cache will be encrypted/decrypted when saving/loading cache.
+ * cache_crypto_callback is enabled in core.compile_model only.
+ * - First value of the struct is encryption function.
+ * - Second value of the struct is decryption function.
+ * @ingroup ov_runtime_cpp_prop_api
+ */
+static constexpr Property<EncryptionCallbacks, PropertyMutability::WO> cache_crypto_callback{"CACHE_CRYPTO_CALLBACK"};
```

Comment:
maybe we need to have aligned names ? cache_encryption_callbacks as we already have `EncryptionCallbacks` ? The same with other names where we have crypto

Reply:
updated.